Tests/AsyncAlgorithmsTests/Performance/ThroughputMeasurement.swift

```diff
@@ -87,6 +87,57 @@ extension XCTestCase {
     }
   }
   
+  public func measureSequenceThroughput<S: AsyncSequence, Output>(firstOutput: @autoclosure () -> Output, secondOutput: @autoclosure () -> Output, _ sequenceBuilder: (InfiniteAsyncSequence<Output>, InfiniteAsyncSequence<Output>) -> S) async where S: Sendable {
+    let metric = _ThroughputMetric()
+    let sampleTime: Double = 0.1
+    
+    measure(metrics: [metric]) {
+      let firstInfSeq = InfiniteAsyncSequence(value: firstOutput(), duration: sampleTime)
+      let secondInfSeq = InfiniteAsyncSequence(value: secondOutput(), duration: sampleTime)
```

Comment:
I just realized that the `duration` property on `InfiniteAsyncSequence` is vestigial from prior evoluation of this code. But we can handle removing that in a separate PR.